src/embind/embind.js

```diff
@@ -36,7 +36,11 @@ var LibraryEmbind = {
   $PureVirtualError: undefined,
   $GenericWireTypeSize: {{{ 2 * POINTER_SIZE }}},
 #if EMBIND_AOT
-  $InvokerFunctions: '<<< EMBIND_AOT_OUTPUT >>>',
+  $InvokerFunctions: '<<< EMBIND_AOT_INVOKERS >>>',
+#if MODULARIZE == 'instance'
+  $embindUpdateExports: '<<< EMBIND_AOT_UPDATE_EXPORTS >>>',
```

Comment:
How about adding some comments here about that `embindUpdateExports` does and when?